generators/client/templates/angular/src/main/webapp/app/_app.module.ts

```diff
@@ -2,21 +2,23 @@
 import { AuthInterceptor } from './blocks/interceptor/auth.interceptor';
 <%_ } %>
 import { NgModule } from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
-import { UIRouterModule } from 'ui-router-ng2';
+import { RouterModule } from '@angular/router';
+import { BrowserModule, Title } from '@angular/platform-browser';
 import { Ng2Webstorage } from 'ng2-webstorage';
 
-import { <%=angular2AppName%>SharedModule } from './shared';
+import { JhiLanguageService } from 'ng-jhipster';
```

Comment:
required only when i18n is used either use an ejs condition block or pass the file through copyJs method when defining in files.
Same goes for every other file importing `JhiLanguageService`.